### PR TITLE
Unify citation strings for quotes and pullquotes.

### PR DESCRIPTION
**Pull request by peterwilsoncc**
See #5401

## Description

Use the same text for the `<cite>` element in both quotes and pullquotes and add translator notes to clarify the purpose.

## How Has This Been Tested?

JavaScript changes, ran `npm test`.

## Screenshots (jpeg or gifs if applicable):
quote_and_pull-quote___ _gutenberg_ _wordpress

## Types of changes

* String reduction
* Translator comments

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code has proper inline documentation.

### CODE REVIEW

File: blocks/library/pullquote/index.js [hunk 1]
```diff
@@ -90,6 +90,7 @@ export const settings = {
 							value: fromRichTextValue( nextValue ),
 						} )
 					}
+					/* translators: the text of the quotation */
```
Comment (gziolo) [inline blocks/library/pullquote/index.js]: I don't think this is the way to go to pass context for the translators. I bet we need to extend [__()](https://github.com/WordPress/gutenberg/blob/master/i18n/index.js#L43) to support context param.  @aduth can you confirm?

**Comment by aduth** [comment]
Can you verify by running a production build (`npm run build`) whether these comments are included in the output of `languages/gutenberg.pot` ?

**Comment by peterwilsoncc** [comment]
@aduth, yes it's included in the generated `gutenberg.pot` file after running the build

Snipped extract

```
# the individual or entity quoted
#: blocks/library/pullquote/index.js:104
#: blocks/library/quote/index.js:228
msgid "Write citation…"
msgstr ""

# the text of the quotation
#: blocks/library/pullquote/index.js:94
#: blocks/library/quote/index.js:214
msgid "Write quote…"
msgstr ""
```

**Review by aduth — APPROVED** [review]
👍

**Pull request merged**